src/main/java/FireboltSqlDialect.java

```diff
@@ -175,9 +183,83 @@ public FireboltSqlDialect() {
                             timeUnitNode.getParserPosition());
                     SqlFloorFunction.unparseDatetimeFunction(writer, call2, "DATE_TRUNC", false);
                     break;
+
+                case BETWEEN:
+                    SqlBetweenOperator sqlBetweenOperator = (SqlBetweenOperator) call.getOperator();
+                    if (sqlBetweenOperator.flag == SqlBetweenOperator.Flag.ASYMMETRIC) {
+                        final SqlWriter.Frame frame =
+                                writer.startList(FRAME_TYPE, "", "");
+                        call.operand(VALUE_OPERAND).unparse(writer, sqlBetweenOperator.getLeftPrec(), 0);
+                        unparseBetweenOperatorRightSide(writer, call, sqlBetweenOperator, frame);
+                    }
+                    else {
+                        super.unparseCall(writer, call, leftPrec, rightPrec);
+                    }
+                    break;
+
                 default:
                     super.unparseCall(writer, call, leftPrec, rightPrec);
             }
         }
     }
+
+    /** Firebolt interval syntax: sign INTERVAL int64 time_unit. */
```

Comment:
Firebolt has support for several dimensions, e.g. `INTERVAL '1 year 2 months 3 days'` is this something Sisense also supports?

Reply:
No